src/app/record/components/top-bar/modals/modal-biography/modal-biography.component.scss-theme.scss

```diff
@@ -0,0 +1,39 @@
+@import '~@angular/material/theming';
+@import 'src/assets/scss/material.orcid-theme.scss';
+@import 'src/assets/scss/material.typography';
+
+@mixin model-biography-theme($theme) {
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+  $warn: map-get($theme, accent);
+  $foreground: map-get($theme, foreground);
+  $background: map-get($theme, background);
+  $config: mat-typography-config();
+
+  hr {
+    border-color: map-get($foreground, 'brand-primary');
+    color: map-get($foreground, 'brand-primary');
+    background: map-get($foreground, 'brand-primary');
+  }
+
+  mat-error {
+    color: map-get($foreground, 'state-warning-dark');
+  }
+
+  mat-hint {
+    color: map-get($foreground, 'text-dark-mid');
+  }
+
```

Comment:
Usea a `src/assets/scss/material.orcid.overwrites.scss/_form-field-theme.scss` to add the material overwrites here

Reply:
Done